htdocs/core/boxes/box_actions.php

```diff
@@ -152,7 +152,7 @@ function loadBox($max=5)
                     );
 
                     $this->info_box_contents[$line][] = array(
-                        'td' => 'align="right" width="18"',
+                        'td' => 'align="right" width="18" class="classfortooltip"',
```

Comment:
What is goal oh having a classfortooltip on a td ? There is no title, so nothing to show on tooltip.